### PR TITLE
feat: update acknowledgements

### PR DESCRIPTION
**Pull request by MalinAhlberg**
This PR addresses [issue #16](https://github.com/NBISweden/SysDev-Maple/issues/16).

The acknowledgment is updated to 

 > This work used the <a href="https://www.egi.eu/infrastructure/cloud/">EGI infrastructure</a> with the support of IN2P3-IRES, INFN-CLOUD-BARI and TR-FC1-ULAKBIM.

### CODE REVIEW

File: proj/pred/templates/pred/help.html [hunk 1]
```diff
@@ -51,9 +51,10 @@ <h3>1. Summary</h3><br>
             peptides in the input sequence(s), thus prior scanning with an separate signal
             peptide predictor is no longer required.
             </p>
-            <P>TOPCONS is supported by the <a href=\"https://www.egi.eu/infrastructure/cloud/\">EGI FedCloud
             </a>with the VO VO.NBIS.SE for computational resource.
```
Comment (nanjiangshu) [inline proj/pred/templates/pred/help.html:54]: This line should be deleted.

**Review by nanjiangshu — COMMENTED** [review]
Looks great @MalinAhlberg , thanks a lot for working on this issue. I hope you weren't be annoyed by the messy code base. 

The acknowledgement to the EGI has been displayed in two places: one in the `help.html` and another at the bottom of `submit_seq.html`.

Maybe a refactoring is needed so that text for the acknowledgement at these two places refer to a unique resource. 

However, the quickest solution might be remove the acknowledgement from the `help.html` and keep only at the bottom of `submit_seq.html`

**Comment by MalinAhlberg** [comment]
> Looks great @MalinAhlberg , thanks a lot for working on this issue. I hope you weren't be annoyed by the messy code base.
> 
> The acknowledgement to the EGI has been displayed in two places: one in the `help.html` and another at the bottom of `submit_seq.html`.
> 
> Maybe a refactoring is needed so that text for the acknowledgement at these two places refer to a unique resource.
> 
> However, the quickest solution might be remove the acknowledgement from the `help.html` and keep only at the bottom of `submit_seq.html`

Sorry for missing that! It's fixed in commit 849cfa2. I think refactoring sounds good, but maybe it's wiser to save that for a larger refactoring issue, which could aim at looking through more parts of the code?

**Comment by nanjiangshu** [comment]
It makes sense @MalinAhlberg, refactoring is not so important and we should actually make do as much MVP as possible for this project unless more funding is provided. 

This line https://github.com/NBISweden/predictprotein-webserver-topcons2/blob/feat/update-acknowledgments/proj/pred/templates/pred/help.html#L54 should be deleted. 

Since you are on leave now, do you mind if I make the commit to this PR?

**Comment by MalinAhlberg** [comment]
Sorry! Really hope I got it right this time. Otherwise, feel free to do any changes you see fit!

**Review by nanjiangshu — APPROVED** [review]
Looks great now!

**Pull request merged**